.github/workflows/wash.yml

```diff
@@ -6,10 +6,10 @@ on:
   pull_request:
     branches: [main]
     paths:
-    - ./Cargo.lock
-    - ./Cargo.toml
-    - ./crates/wash-cli
-    - ./crates/wash-lib
+    - Cargo.lock
+    - Cargo.toml
+    - crates/wash-cli/**
+    - crates/wash-lib/**
```

Comment:
If you leave off the `/**` glob, what's the functional difference? Does it only match against the directory itself and not sub-directories, or what?

Reply:
yes, it does not match the contents of the directory